mb-nft-v2/src/minting.rs

```diff
@@ -47,6 +33,8 @@ impl MintbaseStore {
         metadata_id: Option<U64>,
         royalty_args: Option<RoyaltyArgs>,
         minters_allowlist: Option<Vec<AccountId>>,
+        minting_cap: Option<u32>,
+        last_possible_mint: Option<U64>,
```

Comment:
What is `U64`? Is this a Near type?

Reply:
`struct U64(u64)` with a custom implementation for `serde::Serialize` and `serde::Deserialize`. Same for `U128` and `u128`, where the issue is that such large numbers cannot be processed by JS/JSON properly (max is 2**53 because JS only knows floats). So at the interaction points with frontends, these numbers will be stringified and need to be parsed appropriately.